### ISSUE TITLE
Support increment! and decrement! for atomic counters?

### ISSUE THREAD

**Issue by jessedoyle** [aws/aws-sdk-ruby-record#114]
Hi there - thanks for the awesome library!

I recently encountered a use case for counter columns in  DynamoDB. I was wondering if you'd consider support for [atomic counters](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/WorkingWithItems.html#WorkingWithItems.AtomicCounters) in this library?

I was thinking an API may look something like:

```ruby
record = My::Record.find(pk: 'pk', sk: 'sk')
record.my_counter # => 0
record.increment!(:my_counter) # => 1
record.decrement!(:my_counter) # => 0
```

If you're open to the idea, I'd be happy to take a stab at a PR!

Thanks!

**Comment by mullermp** [comment]
Hey, thanks for opening an issue. Of course we're open to the idea! Atomic counters appear to be a new-ish feature (at least in relation to this gem).

**Comment by jterapin** [comment]
Implemented the feature and published a new gem (v2.8.0).
